packages/babel-parser/test/fixtures/experimental/top-level-await/inside-class-property/output.json

```diff
@@ -0,0 +1,63 @@
+{
+  "type": "File",
+  "start":0,"end":33,"loc":{"start":{"line":1,"column":0},"end":{"line":3,"column":1}},
+  "errors": [
+    "SyntaxError: 'await' is only allowed within async functions (2:6)",
+    "SyntaxError: 'await' is only allowed within async functions (2:6)"
```

Comment:
I wonder why this is reported twice :thinking:

Reply:
This is because the error is raised in duplicate with `checkReserved`. 

https://github.com/babel/babel/blob/6e692092fb40950b3fb38eca716a3ad98d3e57f3/packages/babel-parser/src/parser/expression.js#L2349-L2355

I've fixed with workaround by adding a set called `invalidAwaitErrors` to state at 6d3b9c5. Is there better way?